From c7767f31733349bd1807816159ddc895d62b15cf Mon Sep 17 00:00:00 2001
From: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Date: Fri, 10 May 2013 16:48:36 +0200
Subject: [PATCH 536/715] gpio-rcar: Make the platform data gpio_base field
 signed

The gpio_base field is used to specify the desired GPIO base for the
GPIO controller. The GPIO core can automatically allocate a GPIO number
range when the base is set to -1. To make this possible, make the field
signed.

Signed-off-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Acked-by: Linus Walleij <linus.walleij@linaro.org>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit fe472504fea790fd293d28678bb2cc7c49a2d602)

Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit 45e35129315c656546cd4356c055a0cf71f06f95)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 include/linux/platform_data/gpio-rcar.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/platform_data/gpio-rcar.h b/include/linux/platform_data/gpio-rcar.h
index cc472f6..fb2c718 100644
--- a/include/linux/platform_data/gpio-rcar.h
+++ b/include/linux/platform_data/gpio-rcar.h
@@ -17,7 +17,7 @@
 #define __GPIO_RCAR_H__
 
 struct gpio_rcar_config {
-	unsigned int gpio_base;
+	int gpio_base;
 	unsigned int irq_base;
 	unsigned int number_of_pins;
 	const char *pctl_name;
-- 
1.7.10.4

